client/src/components/squids/SquidList.jsx

```diff
@@ -22,20 +24,25 @@ export const SquidList = () => {
   } else if (squidListQuery.isError) {
     squidListQueryOutput = squidListQuery.error.message
   } else {
-    squidListQueryOutput = squids.results.map((squid, index) => {
-      const hrElement = index > 0 ? <hr /> : ""
-      return (
-        <div key={squid.id}>
-          {hrElement}
-          <SquidTile squid={squid} />
-        </div>
-      )
-    })
+    squidListQueryOutput = squids.results.map((squid) => (
+      <div key={squid.id}>
+        <SquidTile squid={squid} />
+      </div>
+    ))
+  }
+
+  const queryClient = useQueryClient()
+  const useSquidRefetch = () => {
```

Comment:
I wouldn't call this a `use***` function because that infers it's a hook function. Hooks can behave a bit differently (and usually have side effects) as opposed to an event handler. I'd probably call this `onRefetch` if you didn't want to use what's baked into `react-hook-form`